examples/bq-email-exports/main.py

```diff
@@ -0,0 +1,181 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# using SendGrid's Python Library
+# https://github.com/sendgrid/sendgrid-python
+
+import os
+import time
+import datetime
+from sendgrid import SendGridAPIClient
+from sendgrid.helpers.mail import Mail
+from google.cloud import bigquery, storage
+from google.auth import iam, default
+from google.auth.transport import requests
+from google.oauth2 import service_account
+
+
+def credentials():
+    """Gets credentials to authenticate Google APIs.
+
+    Args:
+        None
+
+    Returns:
+        Credentials to authenticate the API.
+    """
+    # Get Application Default Credentials if running in Cloud Functions
+    if os.getenv("IS_LOCAL") is None:
+        credentials, project = default(
+            scopes=["https://www.googleapis.com/auth/cloud-platform"])
+    # To use this file locally set IS_LOCAL=1 and populate env var GOOGLE_APPLICATION_CREDENTIALS
+    # with path to service account json key file
+    else:
+        credentials = service_account.Credentials.from_service_account_file(
+            os.getenv("GOOGLE_APPLICATION_CREDENTIALS"),
+            scopes=["https://www.googleapis.com/auth/cloud-platform"],
+        )
+    return credentials
```

Comment:
All this can be removed once you use constructor with default credentials "Client()"